LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I've started using MIT for some new stuff. We're going to want to be consistent eventually so that we can manage open source contributions "correctly" from a liable PoV